docs/public/install.sh

```diff
@@ -48,13 +48,13 @@ detect_os_arch() {
         OS="unknown-linux"
         if [ "${ARCH}" = "aarch64" ]; then
             ARCH="aarch64"
-            TARGET="${ARCH}-${OS}-gnu"
+            TARGET="${ARCH}-${OS}-musl"
```

Comment:
It looks like both gnu and musl will need to be supported.

https://github.com/tombi-toml/tombi/releases/tag/v0.3.39